web/styles/zulip.css

```diff
@@ -118,6 +118,12 @@ body {
        We expect `resize.js` to update this once UI is initialized. */
     --recent-topics-filters-height: 50px;
 
+    /* Bottom whitespace height of message feed and in recent topics table.
+       This is equal to the max-height of compose box. We expect resize.js to
+       replace it with its pixel calculation since even if `vh` has great support,
+       it can change depending on browser / OS on mobile devices. */
+    --bottom-whitespace-height: 40vh;
```

Comment:
So I'm not entirely sure that it's correct to have this have the same value for recent topics and for the message feed. Maybe it's OK, since we're controlling the value via JS, but it might be cleaner to just have a `recent-topics-bottom-whitespace-height` and a `message-feed-bottom-whitespace-height`.

Reply:
Used `max-unexpanded-compose-height`.